src/pages/tips/usage-of-monorepo.mdx

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
I am not sure if mono-repo comes under GitHub tips, we want to have tips around using GitHub. We can get the contributedBy fix as of now.

Reply:
Delete the monorepo blog, will add some tips soon.